ts/Series/Networkgraph/NetworkgraphSeries.ts

```diff
@@ -193,23 +200,100 @@ class NetworkgraphSeries extends Series {
         NodesComposition.destroy.call(this);
     }
 
+    /**
+     * Networkgraph needs a custom initDataLabels function
+     * to initiate the animation in a different way
+     * than for other series (drawDataLabels is called each frame)
+     * @private
+     */
+    public initDataLabels(this: NetworkgraphSeries): SVGElement {
+        const series = this,
+            dlOptions = series.options.dataLabels,
+            shouldInitialize = series.shouldInitializeDataLabels;
+
+        // if it's the first drawDataLabels() call
+        // then we'll create the SVG group for the first time
+        // & setup the animation from opacity 0 to opacity 1
+        // which lasts based on style.transition property
+        if (shouldInitialize) {
```

Comment:
Following the idea of removing `shouldInitializeDataLabels` property: could you check if we can simply check if `series.dataLabelsGroup` exist? Then simple:
```ts
if (!group) {
  // create group;
  // set opacity;
  // return group;
} else {
  // do something
  // return group;
}
```

Should do the trick